### PR TITLE
DRILL-6347: Inconsistent method name "field".

### PR DESCRIPTION
**Pull request by BruceKuiLiu**
The method is names as "field", but the method is mainly doing appending. So that, rename the method as "append" should be better.

### CODE REVIEW

File: exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/PrelVisualizerVisitor.java [hunk 1]
```diff
@@ -88,10 +88,10 @@ public void endFields() {
     }
 
     public void field(String label, boolean value) {
-      field(label, Boolean.toString(value));
+      append(label, Boolean.toString(value));
     }
 
-    private void field(String label, String value) {
+    private void append(String label, String value) {
```
Comment (arina-ielchiieva) [inline exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/PrelVisualizerVisitor.java]: `appendField`?

Comment (HanumathRao) [reply exec/java-exec/src/main/java/org/apache/drill/exec/planner/physical/visitor/PrelVisualizerVisitor.java]: Thanks Bruce for the changes. As Arina pointed out, appendField would be more meaningful in this context.

**Comment by vrozov** [comment]
IMO, neither `append` or `appendField` is a good choice (otherwise it is necessary to change `startNode/endNode` to `appendStart/EndNode`). It is either `field` or `visitField` and should follow visitor design pattern. It may be also good to change `listField` to the same name used for `Boolean` and `String`.

**Comment by arina-ielchiieva** [comment]
@BruceKuiLiu could you please address @vrozov comments?

**Comment by BruceKuiLiu** [comment]
@vrozov Thanks.

**Comment by vrozov** [comment]
LGTM. Please squash commits.

**Pull request closed without merging**